Sources/Styleguide/PointFreeBaseStyles.swift

```diff
@@ -37,12 +38,43 @@ extension Class {
     public static let opacity50 = CssSelector.class("opacity-50")
     public static let opacity75 = CssSelector.class("opacity-75")
     public enum type {
-      public static let largeTitle = CssSelector.class("pf-large-title")
-      public static let title1 = CssSelector.class("pf-title-1")
-      public static let title2 = CssSelector.class("pf-title-2")
-      public static let title3 = CssSelector.class("pf-title-3")
-      public static let headline = CssSelector.class("pf-headline")
-      public static let subhead = CssSelector.class("pf-subhead")
+      public static let largeTitle =
+        Class.pf.colors.fg.black
+          | Class.type.bold
+          | Class.h1
+          | Class.type.lineHeight(0)
```

Comment:
all of these selectors used to have their styles defined down below, but really we can just define them as the union of styles that we already have from `DesignSystems`.